apps/api/src/app/environments-v1/usecases/construct-framework-workflow/construct-framework-workflow.command.ts

```diff
@@ -1,8 +1,16 @@
 import { EnvironmentLevelCommand } from '@novu/application-generic';
-import { IsDefined, IsObject, IsString } from 'class-validator';
+import { IsDefined, IsObject, IsOptional, IsString } from 'class-validator';
 
 export class ConstructFrameworkWorkflowCommand extends EnvironmentLevelCommand {
   @IsString()
   @IsDefined()
   workflowId: string;
+
+  @IsString()
+  @IsOptional()
+  stepId?: string;
+
+  @IsObject()
+  @IsDefined()
+  controlValues: Record<string, unknown>;
```

Comment:
<!-- Chores are simple tasks that must be done before the subject can be “officially” accepted. Usually, these comments reference some common process. Try to leave a link to the process description so that the reader knows how to resolve the chore.  -->
☑ **chore:** These parameters are no longer needed after the removal of transient previews. Please remove them.‏

Reply:
not true, just the stepId is not needed, the values are still transient